### PR TITLE
add Vmware config

### PR DESCRIPTION
**Pull request by evilnick**
basic instructions for configuring vSphere

fixes #462 #457

### CODE REVIEW

File: src/en/config-vmware.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ vsphere:
+    type: vsphere
+    host: <api-endpoint>
+    user: <some-user>
+    password: <some-password>
+    datacenter: <datacenter-name>
+    external-network: <external-network-name>
+```
+
+The values contained within angle brackets (`<` and `>`) need to be replaced
```
Comment (petermatulis) [inline src/en/config-vmware.md]: Sentence strikes as odd to me. Also seen, albeit a different variation, on the Helion instructions. Newbies may also  just replace the values within the brackets (and keep the brackets).

Maybe standardize and clarify the wording. Suggestion:

"Enter your local settings above (ex: "password: secret")."

It's not completely generic as each provider has its own parameters so some editing is required.

**Comment by petermatulis** [comment]
Otherwise good to go.

File: src/en/config-vmware.md (hunk 1 above)
Comment (evilnick) [inline src/en/config-vmware.md]: yes. part of the problem is that this isn't standardised in the boilerplate files themselves, which is something I guess we will have to fix

**Comment by evilnick** [comment]
ok

**Pull request merged**